lib/flapjack/gateways/jsonapi/metrics_methods.rb

```diff
@@ -30,10 +30,16 @@ def event_queue_length
             redis.llen('events')
           end
 
+          def event_queue
+            { 'length' => redis.llen('events') }
+          end
+
           def processed_events
```

Comment:
As you have it documented in those links makes sense - it just looks like you have the same command from event_queue_length, just wrapped in a hash, making it event_queue.